__tests__/app.test.js

```diff
@@ -108,3 +108,49 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("201: responds with object containing key with article data", () => {
+    const updateVotes = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updateVotes)
+      .expect(201)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article !== undefined).toBe(true);
+        expect(article.hasOwnProperty("votes")).toBe(true);
+      });
+  });
+  test("201: responds with object and and updates vote count", () => {
+    const updateVotes = { inc_votes: -5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updateVotes)
+      .expect(201)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article.votes).toBe(95);
+      });
+  });
+  test("400: response with bad request message and error when passed invalid vote count", () => {
+    const updateVotes = { inc_votes: "five" };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updateVotes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body).toEqual({ msg: "bad request" });
+      });
+  });
+  test("400: responds with error message if article_id is invalid", () => {
+    const updateVotes = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/one")
+      .send(updateVotes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body).toEqual({ msg: "bad request" });
+      });
+  });
+});
```

Comment:
Another test, either a 400/200 if inc_vote key is missing - either respond with an error or make no change